app/controllers/scores_controller.rb

```diff
@@ -10,6 +10,11 @@ def create
     redirect_to users_dashboard_index_path
   end
 
+  def update
```

Comment:
we could change this to a flash.now and have it rendered on the page instead of using a redirect